### PR TITLE
Add given and family names to advanced search

### PR DESCRIPTION
**Pull request by DanielPalafox**
https://trello.com/c/kjrf6tcV/6629-advance-search-doesnt-return-diacritics

### CODE REVIEW

**Comment by DanielPalafox** [comment]
[Advance search doesn't return diacritics](https://trello.com/c/kjrf6tcV/6629-advance-search-doesnt-return-diacritics)

File: src/app/core/search/search.service.ts [hunk 1]
```diff
@@ -56,13 +56,15 @@ export class SearchService {
       const searchParameters = []
       if (escapedParams.firstName) {
         let searchValue = `given-names:${escapedParams.firstName}`
+        searchValue += ` OR given-and-family-names:${escapedParams.firstName}`
         if (escapedParams.otherFields === 'true') {
```
Comment (leomendoza123) [inline src/app/core/search/search.service.ts:60]: @amontenegro @dandineen 

So if we are using `given-names:X OR given-and-family-names:` for given names, and `family-name:X given-and-family-names:X` for family names. 

Isn't it more simple to just use `given-and-family-names` for both name and family name inputs?

Comment (leomendoza123) [reply src/app/core/search/search.service.ts:60]: As you have proposed on slack please use the given-and-family-names for inputs with diacritics

Comment (DanielPalafox) [reply src/app/core/search/search.service.ts:63]: Done!

**Pull request merged**